### PR TITLE
stars_1.0

### PR DESCRIPTION
**Pull request by serjgrm**
- [DEMO LINK](https://serjgrm.github.io/layout_stars/)
- [TEST REPORT LINK](https://serjgrm.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  gap: 4px 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center center;
```
Comment (loralevitska) [inline src/style.css:23]: Combine with [`background` property](https://developer.mozilla.org/en-US/docs/Web/CSS/background)

**Review by loralevitska — CHANGES_REQUESTED** [review]
Good job! There are things that should be fixed:
- check you tests and try to pass all of them
image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+}
```
Comment (etojeDenys) [inline src/style.css:54]: combine in one selector using the comma

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  display: inline-block;
+}
+
+.stars__star:first-child {
+  margin: 2px 2px 0 0;
+}
+
+.stars__star:last-child {
+  margin: 2px 0 0 2px;
+}
```
Comment (etojeDenys) [inline src/style.css:30]:
```suggestion
.stars__star:first-child {
  margin-left: 0;
}

```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  width: 96px;
+  height: 116px;
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin: 2px;
```
Comment (etojeDenys) [inline src/style.css:16]:
```suggestion
  margin-left: 4px;
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Pull request closed without merging**